references/product-recommendations-api/test/integration/features/step_definitions/apickli-gherkin.js

```diff
@@ -0,0 +1,328 @@
+/* jslint node: true */
```

Comment:
this seems to contain a lot of unused apickli DSL extension that isn't used by your integration tests. Please delete the unused parts.

Reply:
Upgraded to @cucumber and apickli 3.x